src/components/Card/index.tsx

```diff
@@ -309,7 +389,8 @@ const BackgroundImage = styled(Image)`
   }
 `
 
-const Card: React.FC<CardPropTypes> = ({
+const Card = ({
```

Comment:
Nice job on trying this out 👍 

Personally I think it's own component is better, I feel like this component becomes too big / hard to read and less maintainable. Maybe we should create a horizontal image card and a vertical image card?

What do you think?